authentication/src/main/java/module-info.java

```diff
@@ -0,0 +1,17 @@
+module life.qbic.authentication {
+  requires org.slf4j;
+  requires java.persistence;
+  requires life.qbic.broadcasting;
+
+  exports life.qbic.authentication.domain.user.concept;
```

Comment:
for the domain you could also just export the complete package since you list all of the subpackages here

Reply:
wild card exports are not possible and would be also bad practise